guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -190,11 +190,12 @@ You can use the xref:Adding_VMware_Images_to_Server_{context}[] section to add t
 If you deploy {Project} with the {SmartProxy} templates feature, you must configure {Project} to recognize hosts' IP addresses forwarded over the X-Forwarded-For HTTP header to serve correct template payload.
 
 For security reasons, {Project} recognizes this HTTP header only from localhost.
-For each individual {SmartProxy}, you must configure a regular expression to recognize hosts' IP addresses.
-From the {ProjectWebUI}, you can do this by navigating to *Administer* > *Settings* > *Provisioning*, and changing the *Remote address* setting.
-From the CLI, you can do this by entering the following command:
+For each individual {SmartProxy}, you must configure a setting to recognize hosts' IP addresses.
+To add an IP address (e.g. 192.0.2.10) or range (e.g. 192.0.2.0/24), use the the following installer command:
```

Comment:
```suggestion
To add an IP address (e.g. 192.0.2.10) or range (e.g. 192.0.2.0/24), use the following installer command:
```
A small typo.

Reply:
Thanks! I also dropped `installer` from the sentence so it's just "command".